src/main.cpp

```diff
@@ -33,6 +33,7 @@
 #include "utilmoneystr.h"
 #include "utilstrencodings.h"
 #include "validationinterface.h"
+#include "wallet/wallet.h"
```

Comment:
Including `wallet.h` here makes the smallest diff but I am not sure if this complies with bitcoin coding guidelines?